libc/cmake/modules/LLVMLibCTestRules.cmake

```diff
@@ -768,7 +768,7 @@ function(add_libc_test test_name)
 endfunction(add_libc_test)
 
 # Tests all implementations that can run on the target CPU.
-function(add_libc_multi_impl_test name suite)
+function(add_libc_multi_impl_test unit_test_only name suite)
```

Comment:
I'm a bit unsure of the change here , can you please check this part out before I expand the usage of it to other tests. 
@nickdesaulniers @michaelrj-google